.github/workflows/debug-kindle-link.yml

```
@@ -0,0 +1,52 @@
+          override: true
+      - name: Install cross
+        run: cargo install cross
+      - name: Build server with verbose link
+        working-directory: ./backend
+        run: |
+          export CROSS_CONTAINER_ENGINE=podman
+          export RUST_FONTCONFIG_DLOPEN=on
+          export FONTCONFIG_NO_PKG_CONFIG=1
+          mkdir -p .cargo
+          cat > .cargo/config.toml << 'EOF'
+[env]
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)